Classes/Util.php

```diff
@@ -274,22 +274,23 @@ public static function getConfigurationFromPageId(
 
         if ($initializeTsfe) {
             self::initializeTsfe($pageId, $language);
-            if (!empty($configurationToUse)) {
-                return self::buildTypoScriptConfigurationFromArray($configurationToUse, $pageId, $language, $path);
+            if (empty($configurationToUse)) {
```

Comment:
Isn't it exactly a request for calling `self::initializeTsfe($pageId, $language);`? In most cases the method is called with false - so I don't think it's a mistake